### DISCUSSION TITLE
Type of secure storage used

### DISCUSSION THREAD

**Discussion by chepchepcirkus** [NiklasMerz/cordova-plugin-fingerprint-aio#354] [Q&A]
Hello,

What kind of secure storage is used by biometric flow in IOS platform,
is the keychain native functionnality ?

Thanks for your work

**Comment by NiklasMerz** [comment]
To my knowledge this is the native iOS Keychain. The current version has been built by someone else and I don't know every detail.

Please check the source code: https://github.com/NiklasMerz/cordova-plugin-fingerprint-aio/blob/master/src/ios/Fingerprint.swift
